tests/tests_password.yml

```diff
@@ -0,0 +1,103 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Ensure that the role is idempotent when changing the sa user password
+  hosts: all
+  vars:
+    mssql_accept_microsoft_odbc_driver_17_for_sql_server_eula: true
+    mssql_accept_microsoft_cli_utilities_for_sql_server_eula: true
+    mssql_accept_microsoft_sql_server_2019_standard_eula: true
+  tasks:
+    - name: Set up MSSQL
+      include_role:
+        name: linux-system-roles.mssql
+      vars:
+        mssql_password: "p@55w0rD"
+        mssql_edition: Evaluation
+
+    #  Otherwise this test fails because mssql-server restarts too often
+    - name: Modify the mssql-server service start limit interval
+      replace:
+        path: /etc/systemd/system/multi-user.target.wants/mssql-server.service
+        regexp: StartLimitInterval.*
+        replace: StartLimitInterval=0
+
+    - name: Modify the mssql-server service start limit burst
+      replace:
+        path: /etc/systemd/system/multi-user.target.wants/mssql-server.service
+        regexp: StartLimitBurst.*
+        replace: StartLimitBurst=0
+
+    - name: Reload service daemon
+      systemd:
+        name: mssql-server
+        state: restarted
+        daemon_reload: yes
+
+    - name: Flush handlers
+      meta: flush_handlers
+
+    - name: >
+        Change the password with default settings.
+        Should report Changed.
+      include_role:
+        name: linux-system-roles.mssql
+      vars:
+        mssql_password: "p@55w0rD11"
+
+    - name: Flush handlers
+      meta: flush_handlers
+
+    - name: Verify settings
+      include_tasks: tasks/verify_settings.yml
+      vars:
+        __verify_mssql_password: "p@55w0rD11"
+
+    - name: Change the IP address setting.
+      include_role:
+        name: linux-system-roles.mssql
+      vars:
+        mssql_ip_address: 127.0.0.1
+
+    - name: Flush handlers
+      meta: flush_handlers
+
+    - name: >
```

Comment:
I think you should use `>-`, otherwise the final newline gets included in the string, which seems a bit unusual for a task `name`.

Reply:
Done.